tests/unit_tests/test_material.py

```diff
@@ -223,6 +223,32 @@ def test_isotropic():
     assert m2.isotropic == ['H1']
 
 
+def test_get_elements():
+    # test that zero elements exist on creation
+    m = openmc.Material()
+    assert len(m.get_elements()) == 0
+
+    # test addition of a single element
+    m.add_element('Li', 0.2)
+    assert 'Li' in m.get_elements()
```

Comment:
Turns out I was only making the review on a single commit and missed this one. My bad :disappointed: 
```suggestion
    assert m.get_elements() == ["Li"]
```